### PR TITLE
(docs)(DOCUMENT-652) fixes metaparameter log level order

### PR DESCRIPTION
**Pull request by jbondpdx**
Also fixes missing details on lookup merge option "knockout_prefix"; tiny doc fixes.

### CODE REVIEW

File: lib/puppet/functions/lookup.rb [hunk 1]
```diff
@@ -96,7 +96,8 @@
 # * `{'strategy' => 'deep', <DEEP OPTION> => <VALUE>, ...}` --- Same as `'deep'`,
 # but can adjust the merge with additional options. The available options are:
 #     * `'knockout_prefix'` (string or undef) --- A string prefix to indicate a
-#     value should be _removed_ from the final result. Defaults to `undef`, which
+#     value should be _removed_ from the final result. If a value is exactly equal
+#     to the prefix, it will knockout the entire element.Defaults to `undef`, which
```
Comment (Magisus) [inline lib/puppet/functions/lookup.rb]: Space missing in "element.Defaults".

**Comment by jbondpdx** [comment]
Oooh, thanks @Magisus , fixed.

**Comment by Magisus** [comment]
Also looks like your first commit is still missing a `(docs)` tag, and your second commit kind of seems to have a misleading message (and the third can probably be squashed into the second). Can you rebase this to fix the prefix and better reflect what the different commits do? Thanks!

**Comment by jbondpdx** [comment]
Yeah, it was messy, sorry @Magisus . I think it's fixed up now.

**Comment by Magisus** [comment]
Looks great, thanks!

**Pull request merged**